src/node.rs

```diff
@@ -65,10 +66,10 @@ impl NodeBuilder {
         }
     }
 
-    /// Override the default min section size.
-    pub fn min_section_size(self, min_section_size: usize) -> Self {
+    /// Override the default network config.
+    pub fn network_cfg(self, network_cfg: Option<NetworkParams>) -> Self {
```

Comment:
I think this can take just `NetworkParams` as you are unlikely to ever need to pass `None` there, as it is the default.